File: BackEnd/BackEnd/Models/Eps.cs
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace BackEnd.Models
{
    public class Eps
    {
        [BsonId]
        [BsonRepresentation(MongoDB.Bson.BsonType.ObjectId)]
        public ObjectId Id { get; set; }

        [BsonElement("codigo")]
        public string Codigo { get; set; }

        [BsonElement("entidad")]
        public string Entidad { get; set; }

        [BsonElement("nit")]
        public string Nit { get; set; }

        [BsonElement("regimen_codigo")]
        public string Regimen_Codigo { get; set; }

        [BsonElement("regimen_descripcion")]
        public string Regimen_Descripcion { get; set; }
    }
}

File: FrontEnd/FrontEnd/src/app/models/eps.ts
export class Eps{
    id : string;
    codigo : string;
    entidad : string;
    nit: string;
    regimen_codigo:string;
    regimen_descripcion:string;

  constructor(
    id: string,
    codigo: string,
    entidad: string,
    nit: string,
    regimen_codigo: string,
    regimen_descripcion: string
) {
    this.id = id
    this.codigo = codigo
    this.entidad = entidad
    this.nit = nit
    this.regimen_codigo = regimen_codigo
    this.regimen_descripcion = regimen_descripcion
  }

}
